packages/legacy-client/src/client/baseTypes/ActionType.ts

```diff
@@ -30,6 +33,11 @@ export type ActionExecutionOptions = {
   returnEdits?: ReturnEditsMode;
 };
 
+export type BulkActionExecutionOptions = {
+  mode?: never;
```

Comment:
Don't want users trying to pass in any sort of execution mode as that isn't supported by the endpoint

Reply:
The problem with this though is it is suggested in intellisense even though it can only be undefined.

It can just be omitted right? https://www.typescriptlang.org/play?#code/KYDwDg9gTgLgBDAnmYcBCBXANgawIIDGMAlhAHYCiIwBGJ5A8mPWQM5wC8cA3gFBxwAthAAmwAPwAuOGWAA3YFADc-OFGAwMUSiOIxWUmRkEAjRXF4BfXmIJYAhurgAzDGSKkyLsgAoIYaUxcQhYqGjpPJhZWAEppOQhiERVeZ18+AQEAeiy4YjIYYCwsYlZgNlRWAAsIAHd2AANhMQarGN5eUEhYBGRUIPwPcjDaFijPVgAmTh5VdU1tCl19QzJjMygLa1sHJ1d3Fm9JvwD0bEHQ6lHI5gnJuLgEpJU044zM5uBpAFYlOByZBB4FhgABzexYOBgRz2QR5ZxwRAQDBwABWGFY8Aggj0bSAA

Reply:
Ah yea you're right, I think I needed this before when we tried the original syntax and I had some conditional types for the options, but yea I think I can just omit now, on it